actions/build/evochamber.py

```diff
@@ -24,6 +24,6 @@ def __init__(self, ai):
         furthest_base = self.ai.townhalls.furthest_to(self.ai.game_info.map_center)
         second_base = (self.ai.townhalls - {furthest_base}).closest_to(furthest_base)
         await self.ai.build(
-            EVOLUTIONCHAMBER, near=second_base.position.towards_with_random_angle(self.ai.game_info.map_center, -10)
+            EVOLUTIONCHAMBER, second_base.position.towards_with_random_angle(self.ai.game_info.map_center, -10)
```

Comment:
make it - 14

Reply:
we can try that, i think this makes it worse.

Reply:
It worked on local test, it puts it on the border and not on the mineral line